waspc/src/Wasp/AppSpec/Api.hs

```diff
@@ -0,0 +1,34 @@
+{-# LANGUAGE DeriveDataTypeable #-}
+
+module Wasp.AppSpec.Api
+  ( Api (..),
+    HttpMethod (..),
+    method,
+    path,
+  )
+where
+
+import Data.Data (Data)
+import Wasp.AppSpec.Core.Decl (IsDecl)
+import Wasp.AppSpec.Core.Ref (Ref)
+import Wasp.AppSpec.Entity (Entity)
+import Wasp.AppSpec.ExtImport (ExtImport)
+
+data Api = Api
+  { fn :: ExtImport,
+    entities :: Maybe [Ref Entity],
+    httpRoute :: (HttpMethod, String),
```

Comment:
Maybe just drop a short comment here saying that snd element in the tuple is route path, describe it a bit?

Btw, it is possible to have a Haskell record here, like you had it before, while on other hand having it as a tuple in Wasp DSL -> we do it with dependencies in AppSpec. I am not saying you have to do it, but it is a way to use syntax we want in DSL while giving it some extra structure in AppSpec when we want.

Reply:
I'll add a short comment just to wrap this PR up. I also have helpers below already that make it feel a bit like record syntax from the outside.